rust/feature-flags/src/api/handler.rs

```diff
@@ -738,4 +787,61 @@ mod tests {
         assert!(!result.error_while_computing_flags);
         assert_eq!(result.feature_flags["test_flag"], FlagValue::Boolean(true));
     }
+
+    #[test]
+    fn test_process_group_property_overrides() {
```

Comment:
wanted to make sure I'm correctly overriding the group values with any passed in property overrides.